docs/source/en/conceptual/stable_diffusion.mdx

```diff
@@ -1,15 +1,332 @@
+                                                                                                                                                                                                                                              
+image = pipe(prompt, generator=generator, num_inference_steps=20).images[0]                                                                                                                                                                   
+image                                                                                                                                                                                                                                         
+```                                                                                                                                                                                                                                           
+                                                                                                                                                                                                                                              
+![img](https://huggingface.co/datasets/diffusers/docs-images/resolve/main/stable_diffusion_101/sd_101_3.png)                                                                                                                                                                                                                                                                                                                                                                                
+                                                                                                                                                                                                                                              
+The image now does look a little different, but it's arguably still of equally high quality. We now cut inference time to just 4 seconds though 😍. 
+
+## Memory Optimization                                                                                                                                                                                                                        
+
+More memory indirectly also means more speed since we're often trying to maximize how many images we can generate per second, so more images per inference, more images per second usually.                                                                                                                                                                                                                                                                                                 
```

Comment:
```suggestion
Less memory used in generation indirectly implies more speed, since we're often trying to maximize how many images we can generate per second. Usually, the more images per inference run, the more images per second too.                                                                                                                                              
```